src/views/index.js

```diff
@@ -467,11 +467,19 @@ const post = ({ msg, aside = false }) => {
     typeof msg.value.content.contentWarning === "string";
 
   const likeButton = msg.value.meta.voted
-    ? { value: 0, class: "liked" }
-    : { value: 1, class: null };
+    ? { value: 0, class: "like liked" }
+    : { value: 1, class: "like" };
 
   const likeCount = msg.value.meta.votes.length;
 
+  const maxLikedNameLength = 16;
```

Comment:
Wasn't sure where to put settings like these, didn't want to leave them as magic numbers.